packages/create-sitecore-jss/scripts/samples.json

```
@@ -0,0 +1,66 @@
+﻿[
+  {
+    "initializers": [
+      "nextjs"
+    ],
+    "args": {
+      "appName": "nextjs-vanilla",
+      "destination": "..\\..\\samples\\nextjs-vanilla",
+      "fetchWith": "GraphQL",
+      "prerender": "SSR",
+      "silent": false
+    }
+  },
+  {
+    "initializers": [
+      "nextjs", "nextjs-styleguide-tracking"
```

Missing 'nextjs-styleguide' here
```suggestion
      "nextjs", "nextjs-styleguide", "nextjs-styleguide-tracking"
```